wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -1710,7 +1710,10 @@ extension ZMUserSession {
                 journal: journal,
                 sessionManager: sessionManager
             ),
-            AppVersionMigration_4_3_0(coreCryptoProvider: coreCryptoProvider)
+            // Disabling migration 4.3.0 because it causes database decryption
+            // issues when the user has multiple accounts.
+            //
+            // AppVersionMigration_4_3_0(coreCryptoProvider: coreCryptoProvider)
```

Comment:
issue: if we re-enable it, it won't run because the app will think it's performed the disabled migration successfully. Wouldn't it be better to remove the migration entirely?

Reply:
no i didn't think too much about it, removing it completly would make sense. I wonder if we need to keep a record of migration though like we do for database changes.

It could be useful in the future to know what was introduced and removed in case we encounter a case where a user did migration X but not Y

Reply:
ok we can keep it commented out, then if we reintroduce it it'll be as a new version.